lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
Mmm...
`basedir` is not the location of .eslintrc file for globally installed ESLint...